lib/manager/npm/post-update/npm.spec.ts

```diff
@@ -1,30 +1,25 @@
-import { exec as _exec } from 'child_process';
 import upath from 'upath';
-
-import { envMock, mockExecAll } from '../../../../test/exec-util';
-import { mocked } from '../../../../test/util';
-import * as _env from '../../../util/exec/env';
-import * as _fs from '../../../util/fs/proxies';
+import { envMock, exec, mockExecAll } from '../../../../test/exec-util';
+import { env, fs } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
 import * as npmHelper from './npm';
 
 jest.mock('child_process');
 jest.mock('../../../util/exec/env');
-jest.mock('../../../util/fs/proxies');
+jest.mock('../../../util/fs');
 jest.mock('./node-version');
 
-const exec: jest.Mock<typeof _exec> = _exec as any;
-const env = mocked(_env);
-const fs = mocked(_fs);
-
 describe('manager/npm/post-update/npm', () => {
   beforeEach(() => {
     jest.resetAllMocks();
     jest.resetModules();
     env.getChildProcessEnv.mockReturnValue(envMock.basic);
+    GlobalConfig.set({ localDir: '' });
```

Comment:
I think we don't need this. Please check other tests too. Please check other tests too for yarn, ...

Reply:
Hi @viceice I've added this because tests didn't work without this line. This line is needed for `exec()` - it thrown an exception complaining that the `path` is undefined (meaning `localDir` is undefined). If you require, I can remove this line, but test wouldn't work, I guess.

Reply:
Ah, ok. Forgot about it. Leave as is. 🤗